src/Core/Grid/Query/CategoryQueryBuilder.php

```diff
@@ -158,6 +159,13 @@ private function getQueryBuilder(array $filters)
                 'c.id_category = cs.id_category AND cs.id_shop = c.id_shop_default'
         );
 
+        $qb->leftJoin(
```

Comment:
On a shop with a lot of products, will this slow significantly the Category listing? 🤔

Reply:
I will add a note to the QA team.